### PR TITLE
Develop graph

### PR DESCRIPTION
**Pull request by ben2collins**

### CODE REVIEW

**Comment by ben2collins** [comment]
@drogers98 Ran this stylesheet through the code cleaner.
Do you think it's ready to be merged into 8.x-1.x.

File: idfive-component-library/src/components/ks/graph/graph.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                .svg-inline--fa {
+                    width: 4em;
+                    height: 4em;
+                }
+            }
+        }
+    }
+
+
+        .hp_graph_bundle {
```
Comment (drogers98) [inline idfive-component-library/src/components/ks/graph/graph.scss]: adjust indentation here please.

**Review by drogers98 — CHANGES_REQUESTED** [review]
adjust one spacing issue and will merge.

File: idfive-component-library/src/components/ks/graph/graph.scss (hunk 1 above)
Comment (ben2collins) [reply idfive-component-library/src/components/ks/graph/graph.scss]: I followed your recommendation and  adjusted the stylesheet's indentation in my latest commit.

**Pull request merged**